scripts/install-android-apk-with-retry.sh

```
@@ -0,0 +1,36 @@
+
+set -eu
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <apk-path>"
+  exit 1
+fi
+
+apk_path="$1"
+max_attempts="${ANDROID_APK_INSTALL_MAX_ATTEMPTS:-5}"
+retry_delay_seconds="${ANDROID_APK_INSTALL_RETRY_DELAY_SECONDS:-5}"
+script_dir="$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)